### ISSUE TITLE
Allow the comparison of two statistics IO parsings

### ISSUE THREAD

**Issue by Jorriss** [Jorriss/StatisticsParser#23]
This feature would allow the user to input two statistics outputs and have the parser compare them. This would mean StatisticsParser would have to be able to parse more than one output at a time. Most likely via tabs.

**Comment by fatherjack** [comment]
Yes, so the statistics of consecutive executions of the same query can be parsed to show reduced (or increased) IO and TIME each step from start to end of a tuning exercise. Indications of improvement can be in text or by colour shading or insertion of summary text below each execution.

**Comment by Jorriss** [comment]
@fatherjack I'm thinking that there's a baseline statistics parsing. Then subsequent parsings will compare against that baseline. Now, how that's all going to look...I have no idea. Well a small idea.

**Comment by CrimsonVex** [comment]
Did this ever eventuate?
